.travis.yml

```diff
@@ -1,62 +1,132 @@
-language: python
-sudo: false
-python:
-  - "3.5"
-  - "3.6"
-cache:
-  pip: true
-  directories:
-    - node_modules
-install:
-  - pip install -U pip setuptools wheel
-  - pip install tox-travis codecov
-  - nvm install 8
-  - npm i
-  - npm run build-assets --production
-  - npm run build-emails
-script:
-  - tox
+---
 env:
   global:
+    - HADOLINT=${HOME}/hadolint
+    - HADOLINT_VERSION=1.11.1
+    - HADOLINT_URL=https://github.com/hadolint/hadolint/releases/download
```

Comment:
I don't think we need `hadolint` here. We don't typically run linters on Travis. It would be nice to get `hadolint` [on Code Climate](https://docs.codeclimate.com/docs/list-of-engines).

Reply:
Yea, I looked into hadolint on codeclimate first and unfortunately, there is no engine yet. 
Thats why I added it to travis. https://github.com/hadolint/hadolint/issues/20

Reply:
I'll track this issue `https://github.com/hadolint/hadolint/issues/20` for now and if hadolint is introduced as an engine, I'll try to make the codeclimate changes then. I was hoping to introduce another stage for yaml linting and helm linting - I am writing some helm charts for this now (gonna try to make PR for that soon) and was hoping to add a yaml lint stage for that. Again codeclimate doesn't support it.

Can you review the rest of the PR and see what you think, I find stages easier to read that the matrices.

Reply:
I like the stage change but I'm not too keen on running linters on Travis. Especially so given that commercial uses of Saleor are more likely to use our CircleCI config.

Reply:
Sure. It's fine. I don't use travis myself. I am writing my own `.gitlab-ci.yml`, I can put it there if i need it. 
Thanks for reviewing.